readme.md

```diff
@@ -8,58 +8,112 @@
 [![Backers][backers-badge]][collective]
 [![Chat][chat-badge]][chat]
 
-[**rehype**][rehype] plugin to wrap a document around a fragment.
+**[rehype][]** plugin to wrap a fragment in a document.
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Install](#install)
+*   [Use](#use)
+*   [API](#api)
+    *   [`unified().use(rehypeDocument[, options])`](#unifieduserehypedocument-options)
+*   [Example](#example)
+    *   [Example: language](#example-language)
+    *   [Example: CSS](#example-css)
+    *   [Example: JS](#example-js)
+    *   [Example: metadata and links](#example-metadata-and-links)
+*   [Types](#types)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Related](#related)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## What is this?
+
+This package is a [unified][] ([rehype][]) plugin to wrap a fragment in a
+document.
+It’s especially useful when going from a markdown file that represents an
+article or so and turning it into a complete HTML document.
```

Comment:
"or so" doesn't quite fit here, it may make sense to replace it or remove it.

```suggestion
article or similar partial document and turning it into a complete HTML document.
```


```suggestion
article and turning it into a complete HTML document.
```